about.html

```diff
@@ -0,0 +1,125 @@
+</head>
+<body>
+  <header class="nav">
+    <div class="nav-ham-icon">
+      <div class="bar bar1"></div>
+      <div class="bar bar2"></div>
+      <div class="bar bar3"></div>
+    </div>
+    <div class="nav-social">
+      <a href="" id="nav-social-icon-1" class="toggle1"><img alt="facebook-social" src="Images/nav-f.png"></a>
+      <a href="" class="toggle1"><img alt="twitter-social" src="Images/nav-t.png"></a>
+      <a href="" class="toggle1"><h6>English</h6></a>
+      <a href="" class="toggle1"><h6>My Page</h6></a>
+      <a href="" id="nav-social-h63" class="toggle1"><h6 >Logout</h6></a>
+    </div>
+    <nav class="nav-bar">
+      <a href="index.html" class="nav-bar-a1 toggle1" ><img alt="logo" class="nav-bar-image" src="Images/footer-logo.png"></a>
+      <a href="about.html" class="toggle1"><h5 class="red-color">About</h5></a>
+      <a href="#program" class="toggle1"><h5>Program</h5></a>
+      <a href="#join" class="toggle1"><h5>Join</h5></a>
+      <a href="#sponsor" class="toggle1"><h5>Sponsor</h5></a>
+      <a href="" class="toggle1"><h5>News</h5></a>
+      <a href="" class="nav-bar-h56 toggle1"><h5 class="red-color">SC Campaign</h5></a>
+    </nav>
+  </header>
+
+  <section class="intro-text">
+    <h3>"Hello! Sharing world"</h3>
+    <h1>SAVE CAUVERY SUMMIT 2022</h1>
+    <p id="para-border">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Esse voluptas quas magnam reprehenderit incidunt! Similique optio in aliquid autem aut voluptate velit nihil iure molestias!</p>
+    <p id="para-place">Please contact us per Email for any further questions about CC Global Summit 2022!</p>
+    <h6 id="para-email">sc2022@savecauvery.com</h6>
+  </section>
+
+  <section class="body" id="program">
+    <h2 id="body-h31">Save Cauvery</h2>
+    <h2 id="body-h32">Global Summit 2022 Logo</h2>
+    <h2 id="body-h34">Save Cauvery Global Summit 2022 Logo</h2>
+    <hr id="body-hr1">
+    <p id="hidden">The logo of Save Cauvery 2022 was decided through the logo competition from 8, June to 7, July</p>
+    <p id="body-para">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Explicabo, dolores non. Deleniti iure vitae blanditiis? Dolore beatae iusto quas maiores, eos aperiam, nulla ducimus accusantium eius consequatur, qui quos autem reprehenderit aliquam perspiciatis ullam.</p>
+    <div class="body-logo-container">
+      <img alt="logo" id="body-logo" src="Images/logo.png">
+    </div>
+    <hr id="body-hr2">
+    <h2 id="body-h33">See the past Save Cauvery</h2>
+    <hr id="body-hr1">
+    <p id="body-para1">Take a look at the last two Save Cauvery Summits which took place in Tamilnadu.</p>  
+    <div class="body-card-container">
+      <div class="body-card">
+        <h2>2020</h2>
+        <h5>Save Cauvery Summit in TamilNadu</h5>
+      </div>
+      <div class="body-card" id="body-card2">
+        <h2>2020</h2>
+        <h5>Save Cauvery Summit in TamilNadu</h5>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] [Optional] I would suggest you should stick to the design as much as possible. please fix the spacing and please increase the font size so your design will stick to the original design.
- [ ] Please stick to the design as much as possible as this is your last review so I am giving it as optional. I would suggest you please implement the same background provide by Microverse
### Your design
![Screenshot 2022-11-25 224858](https://user-images.githubusercontent.com/81727988/204030733-4b16ee43-d6d2-41b1-853a-44e383ca2dfe.png)
### Figma design
![Figma](https://user-images.githubusercontent.com/81727988/204030773-057977bf-0a07-4ca7-83e6-07b3fa5738f1.png)